docs/design/expressions/member_access.md

```diff
@@ -841,6 +842,39 @@ alias X3 = (i32 as Factory).Make;
 alias X4 = i32.((i32 as Factory).Make);
 ```
 
+## Tuple indexing
+
+A tuple indexing expression is of the form:
+
+-   _expression_ `.` _integer-literal_
+-   _expression_ `->` _integer-literal_
```

Comment:
Kate pointed out to me that none of the examples here or in the proposal use this syntax. Could you add some just so that it's clear to folks that anchor on the examples that this is included?

Reply:
Done.